OneSignalSDK/unittest/src/test/java/com/onesignal/InAppMessagingHelpers.java

```diff
@@ -15,7 +15,7 @@
 public class InAppMessagingHelpers {
     public static final String TEST_SPANISH_ANDROID_VARIANT_ID = "d8cc-11e4-bed1-df8f05be55ba-a4b3gj7f";
     public static final String TEST_ENGLISH_ANDROID_VARIANT_ID = "11e4-bed1-df8f05be55ba-a4b3gj7f-d8cc";
-    public static final String ONESIGNAL_APP_ID = "b2f7f966-d8cc-11e4-bed1-df8f05be55ba";
+    public static final String ONESIGNAL_APP_ID = "b4f7f966-d8cc-11e4-bed1-df8f05be55ba";
```

Comment:
Why did this change this `app_id`? Just curious

Reply:
Because we have a check for this id on the initializationChecker under the init method on OneSignal.java and we don't want to test under that condition